docs/posts/flink-quickstart/index.qmd

```diff
@@ -0,0 +1,304 @@
+---
+title: "Ibis goes real-time! Introducing the new Flink backend for Ibis"
+author: "Deepyaman Datta"
+date: "2024-01-29"
+categories:
+    - blog
+    - flink
+    - stream processing
+---
+
+## Introduction
+
+Ibis 8.0.0 marks the official release of the Apache Flink backend for Ibis.
+Flink is one of the most established stream-processing frameworks out there and
+a central part of the real-time data infrastructure at companies like DoorDash,
+LinkedIn, Netflix, and Uber. The Flink backend is also the first streaming
+backend Ibis supports. Follow along as we define and execute a simple streaming
+job using Ibis!
+
```

Comment:
can we get a few examples of use cases to show users why they care before going straight to installations?

Reply:
this would be good to add in